src/handlers/initiateDataRequest.spec.ts

```diff
@@ -34,7 +29,15 @@ jest.mock('../services/validateRequestSource', () => ({
   isSignatureInvalid: jest.fn()
 }))
 
+jest.mock('../services/queue/sendInitiateDataTransferMessage', () => ({
+  sendInitiateDataTransferMessage: jest.fn()
+}))
+
 describe('initate data request handler', () => {
+  beforeEach(() => {
```

Comment:
Purely a preference thing but I prefer the hooks just before the tests and after the vars.